exporter/redis.go

```diff
@@ -199,6 +200,11 @@ func (e *Exporter) initGauges() {
 		Name:      "slowlog_length",
 		Help:      "Total slowlog",
 	}, []string{"addr", "alias"})
+	e.metrics["total_system_memory"] = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
I don't think you need this here, it should automatically get created in `setMetrics()`